examples/nlp/BERTPretrainingTutorial.ipynb

```diff
@@ -143,17 +144,18 @@
    "outputs": [],
    "source": [
     "# Masked Language Modeling Loss\n",
-    "mlm_classifier = nemo_nlp.TokenClassifier(D_MODEL,\n",
+    "mlm_classifier = nemo_nlp.BertTokenClassifier(D_MODEL,\n",
     "                                          num_classes=tokenizer.vocab_size,\n",
-    "                                          num_layers=1,\n",
+    "                                              activation=torch.nn.functional.relu,\n",
```

Comment:
we try not to expose framework-specific code (e.g. torch/tf) on the application script. what do you think @okuchaiev ?

Reply:
below i see you use activation='tanh'. it might be useful to do the same here?